app/Models/User.php

```diff
@@ -402,7 +402,7 @@ public function isModerated(): bool
 
     public function isNew(): bool
     {
-        return Carbon::now()->diffInMonths($this->created_at) < 1;
+        return ((int) Carbon::now()->diffInMonths($this->created_at, true)) < 1;
```

Comment:
It probably isn't necessary to convert to int for comparisons.

Reply:
Updated in latest.